src/index.d.ts

```diff
@@ -148,4 +149,7 @@ declare module 'zx/experimental' {
 
   type StopSpinner = () => void
   export function startSpinner(title: string): StopSpinner
+
+  export const getCtx: AsyncLocalStorage<any>['getStore']
```

Comment:
This is pretty basic. any is not any types at all.

Reply:
Let’s wait for my idea.

Reply:
I just want to see where my own thinking takes me )

Reply:
```js
$.block(($) => {
   $.verbose = false
   fetch('https://example.com')
})````